source/versioning.rst

```diff
@@ -7,16 +7,20 @@ There are currently three 3-D Secure versions, ``1.0.2``, ``2.1.0`` and ``2.2.0`
 This is meant to be a guide to selecting between different 3-D Secure versions.
 
 1. If the :ref:`preauth call <preauth-usage>` returns a :ref:`not_enrolled`, continue with 3DSv1.
-2. When card scheme is Visa, ``acsInfoInd`` is included. This can be used to
-   determine if ACS supports challenges or only attempts.
+2. If ``acsInfoInd`` is included, this informs if ACS supports challenges or
+   only attempts.
 
-   For Mastercard, ``acsInfoInd`` is not available. Mastercard has
-   automatically enrolled all card ranges, without indicating if they were automatically
-   enrolled in ``acsInfoInd``.
+   To select between ``2.1.0`` and ``2.2.0``, use the highest supported version
+   between in ``dsEndProtocolVersion`` and ``acsEndProtocolVersion``, and the
```

Comment:
I think this could be rephrased to ease the understanding :-) The `between in` confuses me.

Reply:
The `in` is superfluous :+1: